open_humans/models.py

```diff
@@ -140,6 +140,8 @@ class Member(models.Model):
         unique=True,
         default=random_member_id)
     seen_pgp_interstitial = models.BooleanField(default=False)
+    password_reset_redirect = models.CharField(max_length=30, default='',
```

Comment:
this produced an error for me on my first test :slightly_smiling_face: URLs can be quite long. maybe this should even be a TextField?